project/Common.scala

```diff
@@ -74,5 +75,9 @@ object Common extends AutoPlugin {
     testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDF"),
     crossScalaVersions := Seq(scala212, scala213),
     mimaReportSignatureProblems := true,
-    scalafmtOnCompile := true)
+    scalafmtOnCompile := true,
+    // https://github.com/akka/akka-grpc/issues/1238
+    PB.protocExecutable := (if (protocbridge.SystemDetector.detectedClassifier() == "osx-aarch_64")
+                              file("/usr/local/bin/protoc")
```

Comment:
This should be configurable. Hopefully ScalaPB can offer a good solution for it.

Reply:
it's configurable by overriding `PB.protocExecutable` - perhaps that is sufficient?

Reply:
Yes, that should be enough. A note/pointer in the Akka gRPC docs would be good.